packages/profile-sync-controller/src/shared/encryption/encryption.ts

```diff
@@ -210,7 +236,7 @@ class EncryptorDecryptor {
       };
     }
 
-    const newSalt = salt ?? randomBytes(SCRYPT_SALT_SIZE);
+    const newSalt = salt ?? new Uint8Array(SCRYPT_SALT_SIZE);
```

Comment:
Can you validate both Scrypt KDFs work with an empty salt size?

Reply:
FUTURE - This is a missing test on mobile, but I would like us to add a test to assert that both scrypt functions generate the exact same encrypted data (to avoid the issue we had early on)

Reply:
> Can you validate both Scrypt KDFs work with an empty salt size?

I CAN! 😄

Reply:
I'd argue that if we want to avoid running into multiple salts we can hardcode a salt here instead of fiddling with the size.
I'm also concerned about the behavior of different libraries when given empty salts